### ISSUE TITLE
1.4.0  kong config db_export error

### ISSUE THREAD

**Issue by cq-z** [Kong/kong#5197]
### Summary
1.4.0 kong config db_export test.yaml -v error , when have keyauth_credentials .
delete ttl = true,    in the file kong/plugins/key-auth/daos.lua   ，it is ok

### Steps To Reproduce

1. add keyauth_credentials
2. kong config db_export 

### Additional Details & Logs

- Kong version (`$ kong version`)  
1.4.0
- Kong debug-level startup logs (`$ kong start --vv`)
```
/usr/local/share/lua/5.1/lyaml/init.lua:126: cannot dump object of type 'userdata'
Error: 
/usr/local/share/lua/5.1/kong/db/declarative/init.lua:278: bad argument #1 to 'write' (string expected, got nil)
stack traceback:
	[C]: in function 'write'
	/usr/local/share/lua/5.1/kong/db/declarative/init.lua:278: in function 'export_from_db'
	/usr/local/share/lua/5.1/kong/cmd/config.lua:41: in function 'cmd_exec'
	/usr/local/share/lua/5.1/kong/cmd/init.lua:87: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:87>
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.1/kong/cmd/init.lua:87: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:44>
	/usr/local/bin/kong:9: in function 'file_gen'
	init_worker_by_lua:49: in function <init_worker_by_lua:47>
```
- Kong error logs (`<KONG_PREFIX>/logs/error.log`)
- Kong configuration (the output of a GET request to Kong's Admin port - see
  https://docs.konghq.com/latest/admin-api/#retrieve-node-information)
- Operating system

**Comment by gszr** [comment]
Thanks for reporting, @cq-z. This issue has been fixed in #5192 and will be part of the upcoming 1.4.1 patch release. In the meantime, feel free to test the fix and report back. Thanks!

**Comment by cq-z** [comment]
thank you
